data/user_data_base.ps2

```diff
@@ -199,6 +209,12 @@ function Get-MyName {
     return $nameTag.Value
 }
 
+function Get-Apps {
+    $instanceId = (New-Object System.Net.WebClient).DownloadString("http://169.254.169.254/latest/meta-data/instance-id")
+    $appsTag =  Get-EC2Tag | Where-Object {$_.ResourceId -eq $instanceId -and $_.Key -eq 'Apps'}
```

Comment:
Could outpost just always allow `alyx-outpost` rather than having to specifically add it in to the apps list?